lib/elixir/lib/file.ex

```diff
@@ -1542,7 +1542,7 @@ defmodule File do
   executes the given function and then reverts back
   to the previous path regardless of whether there is an exception.
 
-  The current working directory is temporarilly set for the BEAM globally. This
+  The current working directory is temporarily set for the BEAM globally. This
```

Comment:
Can you please fix the other one too? :)

Reply:
There's no other one. It was a copy paste, but I've added this one new word with wrong spelling :)

Reply:
Oh!